test/dlc_tests/container_tests/bin/benchmark/run_pytorch_rn50_training_performance_hpu_synthetic

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+PT_VERSION=$(python -c 'import torch; print(torch.__version__)')
+
+HOME_DIR=/test
+LOG_FILE=/test/logs.txt
+LOG_DIR=`dirname $LOG_FILE`
+
+mkdir -p ${HOME_DIR}/artifacts
+mkdir -p ${LOG_DIR}
+
+LINUX_DIST_NAME=`cat /etc/*-release | grep "^NAME=" | sed 's#^.*=##' | tr -d '"'`
+if [ -z "$LINUX_DIST_NAME" ]; then
+    echo Unable to identify Linux distribution
+    exit 1
+elif [ "$LINUX_DIST_NAME" == "Ubuntu" ]; then
+    apt-get update
+    apt-get install -y imagemagick
+elif [ "$LINUX_DIST_NAME" == "Amazon Linux" ]; then
```

Comment:
Is this required? Are we running on AL?

Reply:
Not necessarily, but doesnt hurt to leave it there in case AML2 will be released. (Its matching DLAMI as well)